src/main/java/com/productservice/ServletInitializer.java

```diff
@@ -0,0 +1,13 @@
+package com.productservice;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {

```

Comment:
I think this configuration is required when we want to run Spring boot app from war file. I have never used it, just read here https://www.baeldung.com/spring-boot-servlet-initializer

Are you running product service with embedded tomcat?

What issues do we get if this class is deleted?